decanter_ai_sdk/client.py

```diff
@@ -107,7 +107,7 @@ def train_iid(
         drop_features: List[str] = [],
         evaluator: Optional[Union[RegressionMetric, ClassificationMetric]] = None,
         holdout_table_id: Optional[str] = None,
-        algos: List[IIDAlgorithms] = [
+        algos: List[Union[IIDAlgorithms, str]] = [
```

Comment:
I think it's cleaner to have `Union[List[IIDAlgorithms], List[str]]`. That way you know every element in the list is the same type